Assets/HoloToolkit/UX/Scripts/BoundingBoxes/BoundingBoxGizmoHandle.cs

```diff
@@ -368,14 +368,17 @@ public void OnInputUp(InputEventData eventData)
         public void OnSourceDetected(SourceStateEventData eventData)
         {
         }
+
         public void OnSourceLost(SourceStateEventData eventData)
         {
-            if (eventData.SourceId == inputDownEventData.SourceId)
+            if ((inputDownEventData != null) &&
```

Comment:
event data should never be null.  If it is then, something is being used improperly in another location.

Reply:
I believe it is wrong, but we can address a fix in vNEXT

Reply:
This might be related to those null references in the InputManger you were getting.

events should _never_ be null